src/confluent_kafka/src/confluent_kafka.c

```diff
@@ -1620,7 +1620,10 @@ PyObject *c_Node_to_py(const rd_kafka_Node_t *c_node) {
 
         cfl_PyDict_SetInt(kwargs, "id", rd_kafka_Node_id(c_node));
         cfl_PyDict_SetInt(kwargs, "port", rd_kafka_Node_port(c_node));
-        cfl_PyDict_SetString(kwargs, "host", rd_kafka_Node_host(c_node));
+        if (rd_kafka_Node_host(c_node))
+                cfl_PyDict_SetString(kwargs, "host", rd_kafka_Node_host(c_node));
+        else
+                cfl_PyObject_SetItemString(kwargs, "host", Py_None);
```

Comment:
This function is not present. Please use `PyDict_SetItemString` instead as used in https://github.com/confluentinc/confluent-kafka-python/blob/master/src/confluent_kafka/src/Admin.c#L3198-L3201

Reply:
Build itself is failing due to this.

Reply:
Fixed.